pkgs/applications/audio/puredata/default.nix

```diff
@@ -1,30 +1,31 @@
 { stdenv, fetchurl, autoreconfHook, gettext, makeWrapper
-, alsaLib, libjack2, tk
+, alsaLib, libjack2, tk, fftwSinglePrec
 }:
 
 stdenv.mkDerivation  rec {
   name = "puredata-${version}";
-  version = "0.47-1";
+  version = "0.48-0";
 
   src = fetchurl {
     url = "http://msp.ucsd.edu/Software/pd-${version}.src.tar.gz";
-    sha256 = "0k5s949kqd7yw97h3m8z81bjz32bis9m4ih8df1z0ymipnafca67";
+    sha256 = "0wy9kl2v00fl27x4mfzhbca415hpaisp6ls8a6mkl01qbw20krny";
   };
 
-  patchPhase = ''
-    rm portaudio/configure.in
-  '';
-
   nativeBuildInputs = [ autoreconfHook gettext makeWrapper ];
 
-  buildInputs = [ alsaLib libjack2 ];
+  buildInputs = [ alsaLib libjack2 fftwSinglePrec];
```

Comment:
Why not `fftw`? If `fftwSinglePrec` is preferred, this should be specified in `all-packages.nix` with `fftw = fftwSinglePrec`.

Reply:
@orivej tbh, idk,  I'll change it to fftw.